File: Ejercicio12/src/app/entidades/usuario.ts
export class Usuario{
    private _nombre: string = ""
    private _pass: string = ""


    constructor(nombre: string, pass: string){
        this._nombre = nombre
        this._pass = pass
    }

    public get nombre(): string {
        return this._nombre
    }
    public set nombre(value: string) {
        this._nombre = value
    }

    public get pass(): string {
        return this._pass
    }
    public set pass(value: string) {
        this._pass = value
    }
    
}

File: Ejercicio21/src/main/java/modelo/entidad/Usuario.java
package modelo.entidad;

public class Usuario {

	private int id;
	private String nombre;
	private String pass;
	
	public Usuario() {}
	
	public Usuario(String nombre, String pass) {
		this.nombre = nombre;
		this.pass = pass;
	}
	

	public int getId() {
		return id;
	}

	public void setId(int id) {
		this.id = id;
	}

	public String getNombre() {
		return nombre;
	}

	public void setNombre(String nombre) {
		this.nombre = nombre;
	}

	public String getPass() {
		return pass;
	}

	public void setPass(String pass) {
		this.pass = pass;
	}

	@Override
	public String toString() {
		return "Usuario [id=" + id + ", nombre=" + nombre + ", pass=" + pass + "]";
	}
	
	
	
}